context/app/api/vitessce_confs/assay_confs.py

```diff
@@ -196,6 +243,11 @@ def get_view_config_class_for_data_types(entity, nexus_token):
         return ImagePyramidViewConf(
             entity=entity, nexus_token=nexus_token)
     if "rna" in hints:
+        dag_names = [dag['name']
+                     for dag in entity['metadata']['dag_provenance_list'] if 'name' in dag]
```

Comment:
`'name' in dag` is a tight enough check? Other things couldn't accidentally include that string? Contrived examples: "enamel" or "ornament".

Reply:
Not sure I follow here - this is just to prevent calling dag['name'] when the dag object doesn't have a 'name' attribute (it seems to be optional).